src/Take.Elephant.Sql/StorageBase.cs

```diff
@@ -47,7 +47,8 @@ protected async Task<bool> ContainsAsync(IDictionary<string, object> filterValue
         {
             using (var command = connection.CreateContainsCommand(DatabaseDriver, Table.Schema, Table.Name, filterValues))
             {
-                return (bool)await command.ExecuteScalarAsync(cancellationToken).ConfigureAwait(false);
+                var queryResult = await command.ExecuteScalarAsync(cancellationToken).ConfigureAwait(false);
+                return queryResult is bool result && result;
```

Comment:
It is better to throw an exception instead of silently accepting if the query returns a value different than boolean._

Reply:
Maybe is better to remove this modification and leave just the changed SQL?